src/utility.js

```diff
@@ -410,7 +412,7 @@ function createItem(args, logger, notifier, requestKeys, lambdaContext) {
         break;
       case 'object':
       case 'array':
-        if (arg instanceof Error || (typeof DOMException !== 'undefined' && arg instanceof DOMException)) {
```

Comment:
in my browser `DOMException` is a subclass of `Error` so the `case 'domexception':` should catch it already, but `isError` now catches it because a `DOMException` also has a `name` and a `message`.